index.js

```diff
@@ -0,0 +1,201 @@
+
+
+const redis = require('redis');
+const host = 'localhost';
+const port = 6379;
```

Comment:
These need to be reading from the environment variables

Reply:
Done!